tests/verify/match_to.out

```diff
@@ -1,26 +1,37 @@
-match_to.ml[1:9 ..1:16]: note
+definition of +[1:1 ..1:1]: warning
```

Comment:
Can we fix all the syntax errors within the verify tests at least? Even if it still generates bogus code like `nil` (though ideally it's valid).

Reply:
done